Remotion/Data/DomainObjects.UnitTests/Persistence/MixedStorageProviderTest.cs

```diff
@@ -42,7 +43,7 @@ public void StorageProvidersCanBeMixed ()
     [Test]
     public void MixinsCanOverrideStorageProviderMethods ()
     {
-      using (MixinConfiguration.BuildFromActive().ForClass(typeof(StorageProvider)).Clear().AddMixins(typeof(StorageProviderWithFixedGuidMixin)).EnterScope())
+      using (MixinConfiguration.BuildFromActive().ForClass(typeof(IStorageProvider)).Clear().AddMixins(typeof(StorageProviderWithFixedGuidMixin)).EnterScope())
```

Comment:
can we mix interfaces or should this RdbmsProvider?

Reply:
Interfaces are valid mixin targets, so this works - all tests are green :-)